src/components/Main/Schedule/Home.tsx

```diff
@@ -119,6 +88,37 @@ export const Home = () => {
       )}
     </>
   );
+  function onTabClick(e: React.ChangeEvent, value: number) {
+    if (value <= data.length - 1) {
+      // if not the last tab, open that tab
+      setTabIdx(value);
+      const isOpened = openedTabs.indexOf(value);
+      if (isOpened < 0) setOpenedTabs([...openedTabs, value]); // adds unopened tabs to array. need way to handle lots of tabs
+    } else setIsNewScheduleVisible(true); // if last tab, open dialog to make new schedule
+  }
+
+  function closeDialogHandler(response: any) {
+    // TODO for some reason theres a lot of rerenders for just this alert. nothing visible to client, very low priority
+    setIsNewScheduleVisible(false);
+    if (response.data) setAlert({ message: response.data, status: 'success' }); // response.statusText = "OK", response.status == 200
+  }
+
+  async function onNewScheduleSubmit(
+    scheduleTitle: string,
+    startDate: string,
+    endDate: string,
+    view: string,
+    team: number,
+  ) {
+    await mutateAddSchedule({
+      scheduleTitle,
+      startDate,
+      endDate,
+      view,
+      team,
+      churchId,
+    });
+  }
```

Comment:
yah I'm still for function to be above return statement. functions interact with the state the most, so I think it makes the most sense for it to be right under the state. also functions, like state, is being used directly in the jsx as well, so it makes sense that everything you look for, whether state or functions are in one place.

This is also not a problem here because react/babel, but you generally don't want to write anything after a return statement and this is considered bad practice in writing.
"Warning: unreachable code after return statement"

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Stmt_after_return

Reply:
So even though it's not a problem here, I think we should follow standard writing rules, for our own sake, for new devs sake, and for future sake.